configs/standalone.js

```diff
@@ -100,7 +100,7 @@ module.exports = function(config, optimist) {
         console.log("and suppress this message.\n");
         host = config.host = "127.0.0.1";
     }
-    if (/:/.test(argv.auth) && !isLocalhost && !process.env.C9_HOSTNAME) {
+    if (!/:/.test(argv.auth) && !isLocalhost && !process.env.C9_HOSTNAME) {
```

Comment:
@lennartcl  should this be
```js
var auth = (argv.auth + "").split(":");
if (!auth[1] && !isLocalhost
```
or was this supposed to always warn when not running on localhost?